### PR TITLE
Update variants to read from correct path in Tailwind 3.x

### PR DESCRIPTION
**Pull request by jcayabyab**
As per suggestion from https://github.com/adoxography/tailwind-scrollbar/issues/35#issuecomment-1050724217. The variants has changed in Tailwind 3.x, and as such needs this change to read the value of the variants array.

### CODE REVIEW

**Review by leonnicklas — APPROVED** [review]
Works as expected, thx @JCayabyab! 👍🏼 

@adoxography merge pls.

**Comment by iantearle** [comment]
Please merge. Works as expected.

**Comment by juninholiveira** [comment]
Please! Merge it! Why is it taking so long?

**Comment by kbysport24** [comment]
When is this merge going to happen ?

**Comment by Sarouche92** [comment]
@adoxography Please, can you Merge it!

**Comment by TxMat** [comment]
please merge

**Comment by leonnicklas** [comment]
@alexincore sorry, don't have write access... 

**As a workaround:**

Install this [PR](https://github.com/adoxography/tailwind-scrollbar/pull/36) (by @JCayabyab) via npm until it get's merged officially...

````sh
npm install adoxography/tailwind-scrollbar#pull/36/head
````

**Comment by ilourt** [comment]
@adoxography  could you please merge this PR. Otherwise it prevents to use it fully with tailwind v3

Thanks by advance

**Review by adoxography — APPROVED** [review]
Looks good, though in retrospect, this should really have been a plugin option, not a "variant" (since it's not a variant). I'll merge it as a hotfix for now.

**Pull request merged**
